src/e2e/java/teammates/e2e/cases/AdminNotificationsPageE2ETest.java

```diff
@@ -42,8 +42,8 @@ public void testAll() {
         ______TS("add new notification");
         NotificationAttributes newNotification = NotificationAttributes
                 .builder("placeholder-notif-id")
-                .withStartTime(TimeHelper.parseInstant("2035-04-01T22:00:00Z"))
-                .withEndTime(TimeHelper.parseInstant("2035-04-30T20:00:00Z"))
+                .withStartTime(TimeHelper.parseInstant("2030-04-01T22:00:00Z"))
+                .withEndTime(TimeHelper.parseInstant("2030-04-30T20:00:00Z"))
```

Comment:
This is because only 10 years from current year is available initially:
![image](https://user-images.githubusercontent.com/69516975/163660973-14eb9dfd-404d-4012-ac93-fcc5f6b04bd8.png)

If we keep them as 2035, need to select a future year first and then choose from the range.
![image](https://user-images.githubusercontent.com/69516975/163661042-5f3b7ad2-d049-42df-9565-a92bebdeb2e6.png)

So there are two ways to address this, either to change the year to a value within 10 years from current year, or to add a modification such that the dropdown will select a later value first.

Reply:
I see! In that case, would there be a way to set the year used as the current year + some number of years? A small touch, but might be good for future-proofing

Reply:
I think parsing an absolute string is still preferred as it prevents possible off-by-one errors due to parsing problem. May I ask what is your suggestion for this problem of converting `LocalDateTime` to `Instant`? After some trials, using relatively instant from `LocalDateTime` consistently gives off by 1 nanosecond error during comparision in CI even if nanosecond is set manually (e.g. using `LocalDateTime.of(LocalDate.now().getYear(), 1, 1, 1, 1, 1, 1);`)